.dockerignore

```diff
@@ -0,0 +1,2 @@
+.venv
+Dockerfile
```

Comment:
Ignoring the `Dockerfile` in the `.dockerignore` file is unusual because the `Dockerfile` is essential for building the Docker image. Consider removing this line unless you have a specific reason for ignoring the `Dockerfile`.